PublicMapping/districtbuilder
# Add expandable metrics viewer

## Overview

Adds an expandable metrics viewer, with the ability to pin metrics so they are displayed in the default / collapsed metrics view project sidebar.

### Checklist


### Demo

![image](https://user-images.githubusercontent.com/66973361/122816730-1cd3a000-d2a5-11eb-9546-5a90131c3d1c.png)


### Notes

- Are there any additional metrics that should be included in the viewer? 
- I didn't implement a max pinned metrics length yet, I'm wondering what makes sense to set here @tgilcrest . Also not sure if it makes more sense to implement this client side or server side. Additionally, we will need to create some notifications / error handling around this?
- This will definitely need a style pass @jfrankl - namely, on the CSS animations and on the overlap between the 'Saved' message and the collapse icon

## Testing Instructions

- `./scripts/update`
- Open a project, and click the expand button to open the full screen metrics viewer
- Expect: all available metrics are displayed
- Click the pin icon to pin / unpin a metric, then click the collapse button
- Expect: pinned metrics are displayed

Closes #115
